model/task_queue.go

```diff
@@ -411,20 +411,21 @@ func ClearTaskQueue(distroId string) error {
 	}
 	distroQueueInfo, err = GetDistroAliasQueueInfo(distroId)
 	if err != nil {
-		catcher.Add(errors.Wrap(err, "error getting task alias queue info"))
+		catcher.Add(errors.Wrap(err, "getting task alias queue info"))
```

Comment:
`catcher.Wrap`

Reply:
Missed this.

Reply:
Fixed, and removed the `if err != nil` check.